docs/plugins.rst

```diff
@@ -24,27 +27,32 @@ matches a rule, the plugin may return a response.
 
 Additional methods should be defined on your ``Plugin`` class that will listen and optionally respond to incoming messages. They are registered with the app using one of the following decorators from ``botbotme_plugins.decorators``:
 
-* ``listens_to_mentions(regex)``: A method that should be called only when the bot's nick prefixes the message and that message matches the regex pattern. For example, ``[o__o]: What time is it in Napier, New Zealand?``. The nick will be stripped prior to regex matching.
-* ``listens_to_all(regex)``: A method that should be called on any line that matches the regex pattern.
+* :py:`listens_to_mentions(regex)`: A method that should be called only when the bot's nick prefixes the message and that message matches the regex pattern. For example, ``[o__o]: What time is it in Napier, New Zealand?``. The nick will be stripped prior to regex matching.
+* :py:`listens_to_all(regex)`: A method that should be called on any line that matches the regex pattern.
+* :py:`listens_to_comamnd(cmd)`: A method that should be called on any line that starts with the command prefix, followed by :py:`cmd`. All further arguments are passed through in a list. For example, ``!list ops``.
```

Comment:
Spelling: "comamnd"

Reply:
Jesus Christ, that's bad. Sorry, won't happen again!